### PR TITLE
[AND-3313] [MoPub] Refactor VungleBanner adapter

### PR DESCRIPTION
**Pull request by anton-blashchuk**
updated code to accept different sizes for MREC making it available to play where banner fits
https://vungle.atlassian.net/browse/AND-3313

### CODE REVIEW

**Review by HuFangshuai — CHANGES_REQUESTED** [review]
We also need to refactor this code part on loading ad:
https://github.com/Vungle/mopub-android-mediation/pull/60/files#diff-aceb4d76c06d9382480e8f101358698c3e8c041110a4e1c7d93efd5d88d967a7L138-L145

File: Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java [hunk 1]
```diff
@@ -170,7 +169,7 @@ public void run() {
         }
         final boolean isMRECFormat = "medium_rectangle".equals(adUnitFormat);
         if (isMRECFormat) {
-            if (adWidthInDp >= VUNGLE_MREC.getWidth() && adHeightInDp >= VUNGLE_MREC.getHeight()) {
+            if ((adWidthInDp >= VUNGLE_MREC.getWidth() && adHeightInDp >= VUNGLE_MREC.getHeight())) {
```
Comment (HuFangshuai) [inline Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java]: no need to make changes here.

File: Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java [hunk 2]
```diff
@@ -412,33 +405,7 @@ protected void onVisibilityChanged(@NonNull View changedView, int visibility) {
                                         loadSucceeded = true;
                                         layout.addView(mVungleBannerAd);
```
Comment (HuFangshuai) [inline Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java:406]: For MREC ad size we have to use addView(view, param) to force layout MREC view.

Comment (alexander-gubov) [reply Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java:406]: VungleBanner should handle MREC size as well as any other banner size

File: Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java (hunk 1 above)
Comment (anton-blashchuk) [reply Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java]: that was overlooked :) reverted

**Comment by anton-blashchuk** [comment]
> We also need to refactor this code part on loading ad:
> https://github.com/Vungle/mopub-android-mediation/pull/60/files#diff-aceb4d76c06d9382480e8f101358698c3e8c041110a4e1c7d93efd5d88d967a7L138-L145

updated

**Comment by anton-blashchuk** [comment]
> We also need to refactor this code part on loading ad:
> https://github.com/Vungle/mopub-android-mediation/pull/60/files#diff-aceb4d76c06d9382480e8f101358698c3e8c041110a4e1c7d93efd5d88d967a7L138-L145

@HuFangshuai please check if issue you mentioned is resolved and leave review

**Review by HuFangshuai — APPROVED** [review]
LGTM

**Review by alexander-gubov — APPROVED** [review]

**Pull request merged**